exporter/kafkaexporter/README.md

```diff
@@ -24,8 +24,8 @@ The following settings can be optionally configured:
 - `brokers` (default = localhost:9092): The list of kafka brokers.
 - `resolve_canonical_bootstrap_servers_only` (default = false): Whether to resolve then reverse-lookup broker IPs during startup.
 - `client_id` (default = "sarama"): The client ID to configure the Sarama Kafka client with. The client ID will be used for all produce requests.
-- `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the kafka topic to export to.
-- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic. If `topic_from_attribute` is not set, the message's topic will be set to the value of the configuration option `topic` instead. 
+- `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the default kafka topic to export to. When used in conjunction with `topic_from_attribute` or `topic.WithTopic`, this is the default/fallback topic.
+- `topic_from_attribute` (default = ""): Specify the resource attribute whose value should be used as the message's topic. This option, when set, will take precedence over the default topic and `topic.WithTopic`. If `topic_from_attribute` is not set, the message's topic will be set to the value of the configuration option `topic` instead. 
```

Comment:
`topic.WithTopic` does not seem to be a config option. 

I don't think we have to document it here, but if it is here then it would be better to document it.

Reply:
It's not a config option.
I'm not sure the right approach here.  I added it here to document the order of precedence.

Would you prefer I remove it and not document it, or add documentation in another section so it can be referenced here?

Reply:
either clarify that the `topic.WithTopic` is programmatic API or I would prefer to remove it from there and add note somewhere else.